### PR TITLE
add callback payload information

### PR DESCRIPTION
**Pull request by sanoopsandy**

### CODE REVIEW

File: docs/bank-connect/react.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Once the dependency is added. You can use bank connect view anywhere in your com
 
 `onSuccess` attribute gives callback when document has been successfully uploaded.
 
-`fromDate` & `toDate` attributes will be used to fetch statement for the given time period.
+`fromDate` & `toDate`(OPTIONAL) attributes will be used to fetch statement for the given time period. If not provided default date range is 3 months fro current date.
```
Comment (shouryaps) [inline docs/bank-connect/react.md:41]: "`fro`" should be "`from`" here

File: docs/bank-connect/react.md [hunk 2]
```diff
@@ -49,4 +49,21 @@ Please make sure from date is always less than to date.
...
+`onSuccess` function will return a payload which will contain information from which further data can be extracted. The payload structure of `onSuccess` is as follows:
+
+```js
+const onSuccess = (payload) => {
+    console.log("Payload", payload)
+}
+```
+`payload` will have following structure.
+
+```json
```
Comment (shouryaps) [inline docs/bank-connect/react.md:62]: Can you check if the keys are correct below, also should the keys be in double-quotes for valid json?

Comment (sanoopsandy) [reply docs/bank-connect/react.md:62]: ive changed the json. and keys has been fixed

File: docs/bank-connect/react.md (hunk 1 above)
Comment (sanoopsandy) [reply docs/bank-connect/react.md]: done

**Pull request merged**